stdlib/experimental/prometheus/prometheus.flux

```diff
@@ -1,7 +1,17 @@
 package prometheus
+import "universe" 
 
 // scrape enables scraping of a prometheus metrics endpoint and converts 
 // that input into flux tables. Each metric is put into an individual flux 
 // table, including each histogram and summary value.  
 builtin scrape
 
+// histogramQuantile enables the user to calculate quantiles on a set of given values
+// This function assumes that the given histogram data is being scraped or read from a 
+// Prometheus source. 
+histogramQuantile = (tables=<-, field, quantile) => 
```

Comment:
I wonder if we should remove the `field` parameter to this altogether.  Most of our transformations require the user to do any filtering up front.  This parameter makes it so that the user cannot get a percentile of several fields at once, which they may want to do.  

If the schema of the prometheus data indicated the type of metric in the group key, then it would be useful to filter on it, e.g.: `|> filter(fn: (r) => r.metric_type == "histogram")`, so that we don't try to compute a percentile from non-histogram data.  But we don't have that, unfortunately.

@nathanielc Thoughts on this? ☝️ What are the general principles we should keep in mind with designing an API like this?

Reply:
I agree we should not do the filtering as part of this function. Users may want to compute the quantile of various different fields for example which can be done using correct grouping.